README.rst

```diff
@@ -17,18 +17,151 @@ Requirements: Python >= 3.9.
 
 2. Clone this repository using the following command::
 
-    git clone https://github.com/<your-username>/openwisp2-docs.git
```

Comment:
Let's rename the title of this document to "OpenWISP Documentation".
I renamed the repo to openwisp-docs (from openwisp2-docs).